docs/architecture.md

```diff
@@ -51,9 +51,9 @@ interface Offer {
   aggregate: PieceCID
   // CID of all the segments part of the aggregate
   pieces: PieceCID[]
-  // identifier of the tenant for the storefront `did:web:web3.storage`
-  // spade relies on tenant naming, so we map it here to tenant
-  tenant: string
+  // identifier of the collection for the tenant `did:web:web3.storage`
+  // spade relies on collections to identify different replication constraints.
+  collection: string
```

Comment:
Why not have both?

Reply:
I'd suggest naming things we store under names that makes senses to us, and then mapping them at the points where we interface with spade. This would be "storefront" for us right?

Reply:
This is the bucket that Spade reads from us. In other words, this is the interface...

Reply:
Might be worth to read https://github.com/web3-storage/dealer/blob/main/docs/architecture.md#spade-integration @alanshaw

Reply:
Ah ok got it!